src/buildx/build.ts

```diff
@@ -310,13 +319,16 @@ export class Build {
     return false;
   }
 
-  private static parseSecretKvp(kvp: string): [string, string] {
+  public static parseSecretKvp(kvp: string, redact?: boolean): [string, string] {
```

Comment:
Make this method public so we can use it in build-push-action repo to redact the `secrets` input for build summary case.

Reply:
Opened https://github.com/docker/build-push-action/pull/1351 as follow-up